src/server/raft_client.rs

```diff
@@ -893,8 +905,28 @@ where
         !self.need_flush.is_empty()
```

Comment:
Should we also check the `full_stores` here?

Reply:
No, connections are notified before they are added to `full_stores`.

Reply:
What I mean is to flush the `full_stores` metric.